connectors/migrations/20230621_is_discussion_flag.ts

```diff
@@ -0,0 +1,28 @@
+import { Op } from "sequelize";
```

Comment:
Will make the field non-nullable once this is ran. In the meantime, it doesn't change anything because we treat `null` the same as `false`